game.js

```diff
@@ -0,0 +1,80 @@
+// get choice of computer in random
+function computerChoice() {
+  const options = ["Rock", "Paper", "Scissors"];
+  const choice = options[Math.floor(Math.random() * options.length)];
+  return choice;
+}
+
+// get the result of round
+function playRound(playerSelection, computerSelection) {
+  if (playerSelection == computerSelection) return "draw";
+  else if (
+    (playerSelection == "Rock" && computerSelection == "Paper") ||
```

Comment:
Using string everywhere can cause typo's so prefer to use them as minimum as possible, here you can create a const object for choices and result and then use it again and again.
Ex:
```
const CHOICES = {
  Rock: "Rock",
  Paper: "Paper",
  Scissors: "Scissors"
};

const RESULTS = {
  Draw: "Draw",
  PlayerLost: "You Lost",
  PlayerWon: "You Won"
};

function playRound(playerChoice, computerChoice) {
  if (playerChoice === computerChoice) return RESULTS.Draw;

  switch (playerChoice) {
    case CHOICES.Rock:
      return computerChoice === CHOICES.Scissors ? RESULTS.PlayerWon : RESULTS.PlayerLost;
    case CHOICES.Paper:
      return computerChoice === CHOICES.Rock ? RESULTS.PlayerWon : RESULTS.PlayerLost;
    case CHOICES.Scissors:
      return computerChoice === CHOICES.Paper ? RESULTS.PlayerWon : RESULTS.PlayerLost;
    default:
      throw new Error(`Invalid choice: ${playerChoice}`);
  }
}
```